styles/styles.css

```diff
@@ -439,15 +465,112 @@ main .divider{
     --nav-width: 1240px;
   }
 
-  p, body,a.button:any-link, button  {
+  p,
+  body,
+  a.button:any-link,
+  button {
     font-size: var(--body-font-size-l);
   }
 
-  a.button:any-link, button {
+  a.button:any-link,
+  button {
     border-radius: 12px;
   }
 }
 
+/***************************************************************
+* section: "Optical Design Section"
+***************************************************************/
+
+#design-for-manufacturability {
+  font-size: var(--heading-font-size-xl);
+  font-family: var(--ff-myradpro-regular);
+  padding: 0.2em;
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+}
+
+.optical-design {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding-left: 1.5em;
+  padding-right: 1.5em;
+}
+
+.optical-design .columns > div > div {
+  display: flex;
+  flex-direction: column;
+  padding-left: 3em;
+}
+
+.optical-design div > p:only-child {
+  text-align: center;
+  font-size: var(--heading-font-size-m);
+  margin-bottom: 0;
+  padding-right: 2em;
+  padding-top: 2em;
+}
+
+.optical-design .columns-wrapper {
+  margin: 0 !important;
+  width: 100%;
+}
+
+.optical-design .columns {
+  margin: 0;
+}
+
+.optical-design .columns > div > div > p:first-child {
+  padding-top: 1em;
+  padding-left: 0.5em
+}
+
+.optical-design .columns > div > div > p > span {
+  width: 45px;
+  height: 60px;
```

Comment:
Each of the icons has a different size on it. I used the size 45x60 for all three of them. The other sizes are 32x63 and 47x63

Reply:
@kmurugulla Changed the icon sizes to match zemax.com
ed407de6d2dee557dc8c931af9e08967e66f69d5